Detections/OfficeActivity/MultipleTeamsDeletes.yaml

```diff
@@ -0,0 +1,23 @@
+id: 173f8699-6af5-484a-8b06-8c47ba89b380
+name: Multiple Teams deleted by a single user
+description: |
+  'This detection flags the occurrences of deleting multiple teams within an hour.
+  This data is a part of Office 365 Connector in Azure Sentinel.'
+tactics:
+  - Impact
+relevantTechniques:
+  - T1485
+  - T1489
+query: |
+
+  // Adjust this value to change how many Teams should be deleted before including
+  let max_delete = 3;
+  // Adjust this value to change the timewindow the query runs over
+  let time_window = 1d;
+  OfficeActivity
+  | where OfficeWorkload == "MicrosoftTeams" 
+  | where TimeGenerated > ago(time_window)
+  | where Operation =~ "MemberAdded"
+  | summarize make_set(ItemName) by UserId
```

Comment:
This should include more information and a min and max of the time, also it is not ItemName, it is TeamName that you need when using with the TeamDeleted operation.  Please use - | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), DeletedTeams = make_set(TeamName) by UserId, Operation

Reply:
Update and discounted Operation from summarize as there is single operation.